### DISCUSSION TITLE
Cannot update issues in app, permissions

### DISCUSSION THREAD

**Discussion by beliaev-maksim** [community/community#29114] [Mobile]
I can update labels and assignees in web browser, but get permission error in the app on android
PS, just updated to the latest, no change

![Screenshot_2022-08-11-08-22-32-579_com.github.android.jpg](https://user-images.githubusercontent.com/51964909/184075515-6ccb6eea-6a95-496e-b0e4-b99ccbc0e05c.jpg)

**Comment by jsoref** [comment]
Duplicate of https://github.com/community/community/discussions/3970

**Comment by beliaev-maksim** [comment]
fixed in the latest version on android
